packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.tsx

```diff
@@ -9,13 +9,13 @@ import MinMaxStakeInfo from './min-max-stake-info';
 
 type TBasis = {
     basis: string;
```

Comment:
@kate-deriv is it supposed to be optional? you mentioned it here: https://github.com/kate-deriv/deriv-app/pull/26/files#r1311302301